src/index.ts

```diff
@@ -90,6 +91,17 @@ async function run () {
             { title: 'none', value: null },
           ],
         },
+        {
+          name: 'usePreset',
+          type: context.usePreset ? null : 'select',
+          message: 'Which bundle would you like to install?',
+          initial: 1,
+          choices: [
+            { title: 'Default (Vuetify)', value: 'default' },
+            { title: 'Base (Vuetify, VueRouter)', value: 'base' },
+            { title: 'Essentials (Vuetify, VueRouter, Pinia)', value: 'essentials' },
+          ],
+        },
```

Comment:
This should be moved up before the typescript prompt.

Reply:
It's done on my new commit